cassovary-examples/src/main/java/RandomWalkJava.java

```diff
@@ -91,7 +91,7 @@ public static void main(String[] args) {
   @SuppressWarnings("unchecked")
   private static Map<Integer, Integer> scalaIntsMapToJavaMap(scala.collection.Map<Object, Object> map) {
     Map<Object, Object> javaMap = JavaConverters$.MODULE$.mapAsJavaMapConverter(map).asJava();
-    HashMap<Integer, Integer> javaIntsMap = new HashMap<>();
+    HashMap<Integer, Integer> javaIntsMap = new HashMap<Integer, Integer>();
```

Comment:
Yep. HashMap<>() compiles for Java 7 and up.
And after pull #124 , cassovary works for Java 7 and up.

cc @pankajgupta